lib/SabreZarafa/PrincipalsBackend.php

```diff
@@ -21,7 +21,7 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
- * Project page: <http://github.com/1afa/sabre-zarafa/>
```

Comment:
Please remove the change of the project page. After that, I can merge the changes.